test/distributed/_tensor/test_dtensor_ops.py

```diff
@@ -130,21 +130,15 @@ def wrapped(fn):
     xfail("combinations"),
     xfail("complex"),
     xfail("constant_pad_nd"),
-    xfail("corrcoef"),
     xfail("count_nonzero"),
     xfail("cross"),
     xfail("cummax"),
     xfail("cummin"),
     xfail("cumsum"),
     xfail("cumulative_trapezoid"),
-    xfail("diag"),
-    xfail("diag_embed"),
-    xfail("diagflat"),
-    xfail("diagonal"),
     xfail("diagonal_scatter"),
     xfail("dist"),
     xfail("dot"),
-    xfail("einsum"),
```

Comment:
Just curious does adding the diagonal ops make einsum test succeed? Since einsum could potentially represent so many different linear algebraic operations, I would imagine there are some cases that are not covered.

Reply:
I suspect that einsum was mostly well-supported, but there was just some missing support from the diagonal side.

Reply:
Hi